presto-main/src/main/java/com/facebook/presto/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -440,6 +436,7 @@ private static URI getBufferLocation(RemoteTask remoteTask, OutputBufferId rootB
             }
         });
 
+        StageLinkage stageLinkage = new StageLinkage(fragmentId, parent, childStageExecutions);
```

Comment:
Seems like unnecessary line move.

Reply:
You're right, but looking at it, it does make more sense, since now they are created in input order. I'll put it in a separate commit